tests/Admin/AdminTest.php

```diff
@@ -781,7 +781,7 @@ public function testOnlyOneSubclassNeededToBeActive()
 
     /**
      * @group legacy
-     * @expectedDeprecation Method "Sonata\AdminBundle\Admin\AbstractAdmin::addSubClass" is deprecated since 3.x and will be removed in 4.0.
+     * @expectedDeprecation Method "Sonata\AdminBundle\Admin\AbstractAdmin::addSubClass" is deprecated since 3.30 and will be removed in 4.0.
```

Comment:
This change with a rebase should dissapear

Reply:
Fixed, i will rebase later